### PR TITLE
Feat(UI): Added announcement filter option for activity feeds.

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the adding the announcement option in thread filter dropdown for activity feeds.

#
### Type of change :

- [] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-09 at 12 49 12 PM
Screenshot 2022-08-09 at 12 49 21 PM
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx [hunk 1]
```diff
@@ -40,6 +40,11 @@ export const threadFilterList = [
     value: 'Conversation',
     icon: <SVGIcons alt="" icon={Icons.COMMENT_GREY} width="16px" />,
   },
+  {
+    name: 'Announcement',
+    value: 'Announcement',
+    icon: <SVGIcons alt="" icon={Icons.ANNOUNCEMENT_BLACK} width="16px" />,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx:46]: Can we provide the alt value for these icons?

File: openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx [hunk 2]
```diff
@@ -121,11 +121,15 @@ const DropDownList: FunctionComponent<DropDownListProp> = ({
...
           <UserTag id={item.value as string} name={item.name as string} />
         ) : (
-          <p
-            className={classNames('tw-truncate', widthClass)}
-            title={item.name as string}>
-            {item.name}
-          </p>
+          <>
+            {item.icon}
+            <span className="tw-p-1" />
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx:126]: Can we take a different approach here?

**Review by Sachin-chaurasiya — APPROVED** [review]
Few comments, otherwise LGTM!

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx]: Done

File: openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx]: Done. Also made similar changes for Users component dropdown.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**